multicluster/config/overlays/leader-ns/kustomization.yaml

```diff
@@ -5,12 +5,6 @@ namePrefix: antrea-mc-
 bases:
```

Comment:
Should we just name the directory overlays/leader? Why it is called leader-ns?

Reply:
Because there are 2 leader yaml, one is applied once and another that is namespace specific. This overlay mainly contains config for the namespace specific one. Could you kindly let me know if you still prefer it to be renamed?

Reply:
If we just have one directory for leader now, I feel overlays/leader/ is better. No strong preference.